text/0000-experimental_keywords.md

```diff
@@ -0,0 +1,74 @@
+- Feature Name: `experimental_keywords`
+- Start Date: 2020-04-30
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+This RFC introduce a new way to specify "experimental" keywords. An experimental keyword exposes a fundamental operation of the language and/or compiler _without_ giving that operation its final intended syntax. This allows for user experimentation with the capability introduced by the feature, even on Stable, while allowing the final syntax for the capability to be decided on and stabilized separately from the initial implementation.
+
+An experimental keyword is given with `r#$keyword`, such as:
+* `r#$yeet`
+* `r#$assembly_block`
+* `r#$raw_ref`
+
+# Motivation
+[motivation]: #motivation
+
+It is often the case that we know we want a new ability to be available for users to try out as soon as possible, even before we know what the final syntax for that ability should look like. In the past the compiler has had special macros for this purpose (`try!` and `await!`). This RFC is a continuation of that idea, while also trying to improve the understanding to general users that the ability they're using isn't in its final form.
```

Comment:
nit: `try!` was a normal library macro.  Perhaps you meant `do catch`, which was special syntax using the reserved-but-currently-unused `do` keyword.